themes/dot/exampleSite/content/billing-pricing/_index.en.md

```diff
@@ -1,7 +1,9 @@
 ---
```

Comment:
I'm a bit confused as to why you changed that within exampleSite in the theme folder. Does that affect the website in any way?

Reply:
I will fix this. Not sure how that happened. 😕

Reply:
Done